packages/manager/apps/zimbra/src/pages/dashboard/AutoReplies/AddAutoReply.page.tsx

```diff
@@ -0,0 +1,640 @@
+import React, {
+  useCallback,
+  useContext,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import {
+  Links,
+  LinkType,
+  Subtitle,
+  useNotifications,
+} from '@ovh-ux/manager-react-components';
+import { useTranslation } from 'react-i18next';
+import { useNavigate, useSearchParams } from 'react-router-dom';
+import {
+  OsdsButton,
+  OsdsFormField,
+  OsdsInput,
+  OsdsRadio,
+  OsdsRadioButton,
+  OsdsRadioGroup,
+  OsdsSelect,
+  OsdsSelectOption,
+  OsdsText,
+  OsdsTextarea,
+  OsdsDatepicker,
+  OsdsCheckboxButton,
+} from '@ovhcloud/ods-components/react';
+import {
+  ODS_THEME_COLOR_HUE,
+  ODS_THEME_COLOR_INTENT,
+  ODS_THEME_TYPOGRAPHY_LEVEL,
+  ODS_THEME_TYPOGRAPHY_SIZE,
+} from '@ovhcloud/ods-common-theming';
+import {
+  ODS_BUTTON_VARIANT,
+  ODS_CHECKBOX_BUTTON_SIZE,
+  ODS_INPUT_SIZE,
+  ODS_INPUT_TYPE,
+  ODS_LOCALE,
+  ODS_RADIO_BUTTON_SIZE,
+  ODS_SPINNER_SIZE,
+  ODS_TEXT_LEVEL,
+  ODS_TEXT_SIZE,
+} from '@ovhcloud/ods-components';
+import { ApiError } from '@ovh-ux/manager-core-api';
+import { useMutation } from '@tanstack/react-query';
+import { ShellContext } from '@ovh-ux/manager-react-shell-client';
+import {
+  useAccount,
+  useAccountList,
+  useDomains,
+  useGenerateUrl,
+} from '@/hooks';
+import {
+  ACCOUNT_REGEX,
+  checkValidityField,
+  checkValidityForm,
+  FormTypeInterface,
+  makeDateFromDDMMYYYY,
+} from '@/utils';
+import Loading from '@/components/Loading/Loading';
+
+export enum AutoReplyTypes {
+  LINKED = 'linked',
+  FREE = 'free',
+}
+
+export enum AutoReplyDurations {
+  TEMPORARY = 'temporary',
+  PERMANENT = 'permanent',
+}
+
+const durationChoices = [
+  {
+    value: AutoReplyDurations.TEMPORARY,
+    key: 'zimbra_auto_replies_add_duration_temporary',
+  },
+  {
+    value: AutoReplyDurations.PERMANENT,
+    key: 'zimbra_auto_replies_add_duration_permanent',
+  },
+];
+
+export default function AddAutoReply() {
+  const { t } = useTranslation('autoReplies/add');
+  const navigate = useNavigate();
+  const { addError, addSuccess } = useNotifications();
+  const context = useContext(ShellContext);
+  const locale = context.environment.getUserLocale();
+  const [searchParams] = useSearchParams();
+  const params = Object.fromEntries(searchParams.entries());
+  const organizationId = searchParams.get('organizationId');
+  const editEmailAccountId = searchParams.get('editEmailAccountId');
+  const [isFormValid, setIsFormValid] = useState(false);
+  const [selectedOrganizationId, setSelectedOrganizationId] = useState(
+    organizationId,
+  );
+
+  const goBackUrl = useGenerateUrl('..', 'path', params);
+
+  const goBack = () => {
+    return navigate(goBackUrl);
+  };
```

Comment:
```suggestion
  const goBack = () => navigate(goBackUrl);
```
We can use the short way.